docs/examples/apim.md

```diff
@@ -111,6 +111,54 @@ provider:
       exposeHeaders:
         - "*"
 
+    # JWT validation APIM policy
+    jwtValidate:
+      headerName: authorization
+      scheme: bearer
+      failedStatusCode: 401
+      failedErrorMessage: "Authorization token is missing or invalid"
+      openId:
+        metadataUrl: "https://path/to/openid/metadata/config"
+      audiences:
+        - "audience1"
+        - "audience2"
+      issuers:
+        - "https://path/to/openid/issuer"
+
+    # Header validation APIM policy
+    checkHeaders:
+      - headerName: x-example-header-1
+        failedStatusCode: 400
+        failedErrorMessage: Not Authorized
+        values: # List of allowed values, otherwise returns error code/message
+          - value1
+          - value2
+      - headerName: x-example-header-2
+        failedStatusCode: 403
+        failedErrorMessage: Forbidden
+        values: # List of allowed values, otherwise returns error code/message
+          - value1
+          - value2
+
+    # IP Validation APIM policies
+    ipFilters:
```

Comment:
is this `ipFilter` or `ipFilters`?

I only see ip-filter here https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#RestrictCallerIPs

Reply:
The xml generated name is `ip-filter`.  The yaml config sections for `ipFilter` and `ipFilters` are for backwards compabibility.  The `ipFilter` previously existed but only supported a single configuration.  `ipFilters` is new and supports 1-to-many configuration.  An example for this would be to apply both an `allow` config as well as a `forbid` config.  We also have the option to just remove the original `ipFilter` setting to avoid confusion but this would be a breaking change to anyone that may be using it.